components/Avatar/index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="avatar.css">
+    <link rel="stylesheet" href="/components/sideNav/sideNav.css">
+  
+    <title>Avatar</title>
+</head>
+<body>
+    <nav class='navigation-bar' >
+        <div class='list-parent-left' id='symbol'>
+            <a class='li-link'href='../../index.html'> <h3>Chikku<span style='color:rgb(10, 177, 10)'>Ui</span></h3> </a>  
+        </div>
+        <div class='list-parent-right' id='modal'>
+            <a  href="javascript:void(0);" class="li-icon" onclick="modelFunction()">
+                <i class="fa fa-bars"></i>
+              </a>
+              <a class='li-link' href='../../index.html'> Home</a>
+              <a class='li-link' href='index.html'>Documentation</a>
+              <a class='li-link' target='_blank' href='https://github.com/vineeth-t/Main/tree/Devlopment'>Github</a>
+              <!-- <a class='li-link'href='#'>V0.1</a> -->
```

Comment:
unused code, you can remove this

Reply:
I have removed the unused code. Thank you for reminding of this